sdk/keyvault/azkeys/crypto/constants.go

```diff
@@ -38,23 +47,50 @@ func PossibleWrapAlgorithmValues() []WrapAlgorithm {
 // EncryptionAlgorithm - algorithm identifier
 type EncryptionAlgorithm string
 
-const (
-	EncryptionAlgorithmA128CBC    EncryptionAlgorithm = "A128CBC"
-	EncryptionAlgorithmA128CBCPAD EncryptionAlgorithm = "A128CBCPAD"
-	EncryptionAlgorithmA128GCM    EncryptionAlgorithm = "A128GCM"
-	EncryptionAlgorithmA128KW     EncryptionAlgorithm = "A128KW"
-	EncryptionAlgorithmA192CBC    EncryptionAlgorithm = "A192CBC"
-	EncryptionAlgorithmA192CBCPAD EncryptionAlgorithm = "A192CBCPAD"
-	EncryptionAlgorithmA192GCM    EncryptionAlgorithm = "A192GCM"
-	EncryptionAlgorithmA192KW     EncryptionAlgorithm = "A192KW"
-	EncryptionAlgorithmA256CBC    EncryptionAlgorithm = "A256CBC"
-	EncryptionAlgorithmA256CBCPAD EncryptionAlgorithm = "A256CBCPAD"
-	EncryptionAlgorithmA256GCM    EncryptionAlgorithm = "A256GCM"
-	EncryptionAlgorithmA256KW     EncryptionAlgorithm = "A256KW"
-	EncryptionAlgorithmRSA15      EncryptionAlgorithm = "RSA1_5"
-	EncryptionAlgorithmRSAOAEP    EncryptionAlgorithm = "RSA-OAEP"
-	EncryptionAlgorithmRSAOAEP256 EncryptionAlgorithm = "RSA-OAEP-256"
-)
+// EncryptionAlgorithmA128CBC is the 'A128CBC' encryption algorithm
```

Comment:
Is there a reason to get rid of all these `const` blocks, other than just not liking `const` blocks? 😸

Reply:
Is this because the tool is choking on the block?  If so, we should fix the tool instead (same for `var` blocks).